app/tests/components_tests/test_tasks.py

```diff
@@ -422,3 +425,22 @@ def test_add_file_to_object(settings, object_type):
         interface_pk=ci.pk,
     )
     assert ComponentInterfaceValue.objects.filter(interface=ci).count() == 1
+
+
+@pytest.mark.parametrize(
+    "container_image_file",
+    ("hello-scratch-oci.tar.gz", "hello-scratch-docker-v2.tar.gz"),
+)
+@pytest.mark.django_db
+def test_get_image_config_and_sha256(container_image_file):
```

Comment:
Should there be tests for all the ValidationErrors raised by that and the related methods?

Reply:
I don't think that it's worth creating corrupt container image files for this.

Reply:
Fair enough!